src/modules/rover_mecanum/module.yaml

```diff
@@ -0,0 +1,201 @@
+module_name: Rover Mecanum
+
+parameters:
+  - group: Rover Mecanum
+    definitions:
+
+      RM_MAN_YAW_SCALE:
+        description:
+          short: Manual yaw rate scale
+          long: |
+            In manual mode the setpoint for the yaw rate received from the rc remote
+            is scaled by this value.
+        type: float
+        min: 0.01
+        max: 1
+        increment: 0.01
+        decimal: 2
+        default: 1
+
+      RM_WHEEL_TRACK:
+        description:
+          short: Wheel track
+          long: Distance from the center of the right wheel to the center of the left wheel
+        type: float
+        unit: m
+        min: 0.001
+        max: 100
+        increment: 0.001
+        decimal: 3
+        default: 0.5
+
+      RM_MAX_YAW_RATE:
+        description:
+          short: Maximum allowed yaw rate for the rover.
+          long: |
+            This parameter is used to cap desired yaw rates and map controller inputs to desired yaw rates in acro mode.
+        type: float
+        unit: deg/s
+        min: 0.01
+        max: 1000
+        increment: 0.01
+        decimal: 2
+        default: 90
+
+      RM_MAX_THR_YAW_R:
```

Comment:
Can't this value actually be calculated from RM_MAX_THRTL_SPD and the rover geometry?

Reply:
okay I see, thanks for the explanation.